### ISSUE TITLE
v1.4.0 Tail plugin - path_key expect boolean value

### ISSUE THREAD

**Issue by schod** [fluent/fluent-bit#2050]
## path_key expect boolean value

When you configure param **path_key** in Tail plugin, you get this error:
`[error] [config map] invalid value for boolean property 'path_key=source_path'`

**path_key description** from Tail [manual page](https://docs.fluentbit.io/manual/pipeline/inputs/tail)

If enabled, it appends the name of the monitored file as part of the record. The value assigned becomes the key in the map.

**To Reproduce**
- Config:
```
[INPUT]
    Name        tail
    Path        /var/log/syslog
    Path_Key source_path

[OUTPUT]
    Name   stdout
    Match  *
```
- Error:
```
[error] [config map] invalid value for boolean property 'path_key=source_path'
```

**Environment**

* Version used: 1.4.0
* Configuration:
```
[INPUT]
    Name        tail
    Path        /var/log/syslog
    Path_Key source_path

[OUTPUT]
    Name   stdout
    Match  *
```
* Operating System and version: Ubuntu 18.04
* Filters and plugins: Tail

**Comment by edsiper** [comment]
thanks for reporting this. We will push a new release with the fix.

**Comment by edsiper** [comment]
fixed by 27f2036a
